### PR TITLE
Bugfix (src/Forms/MultipleReplace.cs)

### PR DESCRIPTION
**Pull request by ivandrofly**
Check this out /cc @niksedk

### CODE REVIEW

File: src/Forms/MultipleReplace.cs [hunk 1]
```diff
@@ -453,6 +454,7 @@ private void moveUpToolStripMenuItem_Click(object sender, EventArgs e)
 
         private void SwapReplaceList(int index, int index2)
         {
+            listViewReplaceList.ItemChecked -= ListViewReplaceListItemChecked;
             bool enabled = listViewReplaceList.Items[index].Checked;
```
Comment (ivandrofly) [inline src/Forms/MultipleReplace.cs]: Actually this is one of the reason why <kbn>CTRL</kbn>+<kbn>Up</kbn> is slow

File: src/Forms/MultipleReplace.cs [hunk 2]
```diff
@@ -32,6 +31,12 @@ internal ReplaceExpression(string findWhat, string replaceWith, string searchTyp
             }
         }
 
+        private const string MultipleSearchAndReplaceItem = "MultipleSearchAndReplaceItem";
+        private const string @Enabled = "Enabled";
```
Comment (niksedk) [inline src/Forms/MultipleReplace.cs]: Could you name this "RuleEnabled" or something?

**Comment by niksedk** [comment]
Looks nice :)

**Comment by ivandrofly** [comment]
Squashed

File: src/Forms/MultipleReplace.cs [hunk 3]
```diff
@@ -591,6 +594,7 @@ private void buttonRemoveAll_Click(object sender, EventArgs e)
         {
             listViewReplaceList.Items.Clear();
             Configuration.Settings.MultipleSearchAndReplaceList.Clear();
+            GeneratePreview();
```
Comment (niksedk) [inline src/Forms/MultipleReplace.cs]: Also nice fix :)

**Pull request merged**